cmd/repo-operations.go

```diff
@@ -104,6 +107,10 @@ func executeCommand(config *GitXargsConfig, repositoryDir string, repo *github.R
 
 	stdoutStdErr, err := cmd.CombinedOutput()
 
+	logger.WithFields(logrus.Fields{
+		"CombinedOutput": string(stdoutStdErr),
+	}).Debug("Received output of command run")
```

Comment:
This is the real fix. It moves the log statement _before_ the error check to ensure we always get log output.